### PR TITLE
feat: Add webhooks on integration tests

### PR DESCRIPTION
**Pull request by gfyrag**

### CODE REVIEW

File: tests/integration/suite/webhooks-configs.go [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+	. "github.com/formancehq/stack/tests/integration/internal"
+	webhooks "github.com/formancehq/webhooks/pkg"
+	"github.com/formancehq/webhooks/pkg/security"
+	"github.com/google/uuid"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Given("empty environment for webhooks configs", func() {
+	When("inserting 0 config", func() {
```
Comment (gfyrag) [inline tests/integration/suite/webhooks-configs.go:20]: Useless block.

File: tests/integration/suite/webhooks-configs.go [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+				},
+			}
+			insertResp, httpResp, err = Client().WebhooksApi.
+				InsertConfig(TestContext()).ConfigUser(cfg).Execute()
+			Expect(err).To(BeNil())
+			Expect(httpResp.StatusCode).To(Equal(http.StatusOK))
+			Expect(insertResp.Data.Endpoint).To(Equal(cfg.Endpoint))
+			Expect(insertResp.Data.EventTypes).To(Equal(cfg.EventTypes))
+			Expect(insertResp.Data.Active).To(BeTrue())
+			Expect(time.Until(insertResp.Data.CreatedAt)).To(BeNumerically("<", time.Second))
```
Comment (gfyrag) [inline tests/integration/suite/webhooks-configs.go:53]: Avoid this. It can easily fail.

File: tests/integration/suite/webhooks-configs.go [hunk 3]
```diff
@@ -0,0 +1,344 @@
...
+			Expect(time.Until(insertResp.Data.UpdatedAt)).To(BeNumerically("<", time.Second))
+			_, err = uuid.Parse(insertResp.Data.Id)
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		Context("getting all configs without filters", func() {
```
Comment (gfyrag) [inline tests/integration/suite/webhooks-configs.go:59]: Find a way to factorize common setup for common tests.
You can use https://onsi.github.io/ginkgo/#table-specs for example

File: tests/integration/suite/webhooks-configs.go [hunk 4]
```diff
@@ -0,0 +1,344 @@
...
+				Expect(resp.Data.Active).To(BeFalse())
+
+				resp, httpResp, err = Client().WebhooksApi.
+					ActivateConfig(TestContext(), insertResp.Data.Id).Execute()
+				Expect(err).NotTo(HaveOccurred())
+				Expect(httpResp.StatusCode).To(Equal(http.StatusOK))
+				Expect(resp.Data.Active).To(BeTrue())
+			})
+
+			Context("getting all configs", func() {
```
Comment (gfyrag) [inline tests/integration/suite/webhooks-configs.go:153]: Useless block.

File: tests/integration/suite/webhooks-configs.go [hunk 5]
```diff
@@ -0,0 +1,344 @@
...
+					InsertConfig(TestContext()).ConfigUser(cfg).Execute()
+				Expect(err).To(BeNil())
+				DeferCleanup(func() {
+					httpServer.Close()
+				})
+			})
+
+			Context("testing the inserted one", func() {
+				It("should return a successful attempt", func() {
+					attemptResp, httpResp, err := Client().WebhooksApi.
```
Comment (gfyrag) [inline tests/integration/suite/webhooks-configs.go:294]: Use eventually for async checks.

File: tests/integration/suite/webhooks-configs.go [hunk 6]
```diff
@@ -0,0 +1,344 @@
...
+				Expect(httpResp.StatusCode).To(Equal(http.StatusOK))
+			})
+		})
+	})
+
+	When("inserting 1 config", func() {
+		secret := webhooks.NewSecret()
+		var insertResp *formance.ConfigResponse
+		BeforeEach(func() {
+			var err error
```
Comment (gfyrag) [inline tests/integration/suite/webhooks-configs.go:37]: Can you use block style déclaration for multiple variables please.

File: tests/integration/suite/webhooks-configs.go (hunk 1 above)
Comment (antoinegelloz) [reply tests/integration/suite/webhooks-configs.go:20]: It's a reminder for context, I don't think this is useless

File: tests/integration/suite/webhooks-configs.go (hunk 2 above)
Comment (antoinegelloz) [reply tests/integration/suite/webhooks-configs.go:53]: What would you do instead?

**Pull request closed without merging**